tests/helpers/test_template.py

```diff
@@ -669,22 +669,19 @@ def test_random_every_time(test_choice, hass):
 
 def test_passing_vars_as_keywords(hass):
     """Test passing variables as keywords."""
-    assert template.Template("{{ hello }}", hass).async_render(hello=127) == "127"
+    assert template.Template("{{ hello }}", hass).async_render(hello=127) == 127
 
 
 def test_passing_vars_as_vars(hass):
     """Test passing variables as variables."""
-    assert template.Template("{{ hello }}", hass).async_render({"hello": 127}) == "127"
+    assert template.Template("{{ hello }}", hass).async_render({"hello": 127}) == 127
 
 
 def test_passing_vars_as_list(hass):
     """Test passing variables as list."""
-    assert (
-        template.render_complex(
-            template.Template("{{ hello }}", hass), {"hello": ["foo", "bar"]}
-        )
-        == "['foo', 'bar']"
-    )
+    assert template.render_complex(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)